modules/REST/API.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\REST;
+
+use PLL_Model;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Setup the REST API endpoints and filters.
+ *
+ * @since 3.7
+ *
+ * @phpstan-import-type Fields from V2\Languages as RestFields
+ * @phpstan-template T of RestFields
```

Comment:
This is because of the stubs:
```php
    /**
     * @phpstan-template T of array
     * @phpstan-implements ArrayAccess<key-of<T>, value-of<T>>
     */
    class WP_REST_Request implements \ArrayAccess
    {
```

Reply:
OK, it was necessary because `V2\Languages` was declaring the template at the class level:
```
 ------ ---------------------------------------------------------------------------------------------------------------------------------------- 
  Line   modules/REST/API.php                                                                                                                    
 ------ ---------------------------------------------------------------------------------------------------------------------------------------- 
  :28    Property WP_Syntex\Polylang\REST\API::$languages with generic class WP_Syntex\Polylang\REST\V2\Languages does not specify its types: T  
 ------ ----------------------------------------------------------------------------------------------------------------------------------------
```
But since it is declared at the method level now, it is useless.
Removed in [273acf3](https://github.com/polylang/polylang/pull/1505/commits/273acf30cb40711424df79dd965860c5713ff258).